src/GO/Scheduler.php

```diff
@@ -126,6 +126,10 @@ public function php($script, $bin = null, $args = [], $id = null)
         $bin = $bin !== null && is_string($bin) && file_exists($bin) ?
             $bin : (PHP_BINARY === '' ? '/usr/bin/php' : PHP_BINARY);
 
+        if (PHP_OS_FAMILY == 'Windows' && stripos($bin, ' ') !== false) {
+            $bin = '"{$bin}"';
```

Comment:
This is correct way that should be applied on all parts of command

```suggestion
            $bin = escapeshellarg($bin);
```